scipy/sparse/csgraph/_shortest_path.pyx

```diff
@@ -488,6 +490,8 @@ def dijkstra(csgraph, directed=True, indices=None,
     predecessors : ndarray, shape ([n_indices, ]n_nodes,)
```

Comment:
```suggestion
    predecessors : ndarray, shape (n_indices, n_nodes,)
```
?

Reply:
Ah! yes.

Reply:
However, I think this is only applicable to functions which accept `min_only` argument because that's where, `n_indices` ignored.

Reply:
Ok makes sense let's leave it as is